docs/ADDITIONAL_INFORMATION.md

```diff
@@ -0,0 +1,14 @@
+Additional Information
+## Release cadence
+<release cadence>
+
+## Versioing
+* This product releases follow [semantic versioning](https://semver.org/).
+
+## Deprecation
+* <Product> deprecation cycle is aligned with [Ansible](https://docs.ansible.com/ansible/latest/dev_guide/module_lifecycle.html).
+
+<Any other Aditional informations, 
+E.g. 
+Enabling/Ignoring SSL certificate validation, 
+release cadence, versioning>
```

Comment:
Having examples that are already in the document doesn't illustrate what other types of things might be added. Please delete "release cadence, versioning" and, if there are other example possibilities you have in mind, add those instead. (Otherwise it is okay to end this with just the SSL certificate validation example.)

Reply:
removed as suggested